torch/distributed/pipelining/stage.py

```diff
@@ -173,6 +173,16 @@ def __init__(
                 f"Pipeline group size {self.group_size} cannot be larger than number of stages {self.num_stages}"
             )
 
+        def stage_global_rank(peer_rank):
```

Comment:
Note: just moved this function from a derived class up to the base class, so it can be used from everywhere.